### ISSUE TITLE
[bug] Odd file issue

### ISSUE THREAD

**Issue by jgx0** [foo-dogsquared/hugo-theme-arch-terminal#18]
so whenever I run hugo server, my site works fine,  but on netlify it doesnt show any of the css. the url is https://unixgang.netlify.app . Running hugo server also gives this error 
```
`WARN 2020/12/18 10:16:08 .File.Dir on zero object. Wrap it in if or with: {{ with .File }}{{ .Dir }}{{ end }}
```

**Comment by foo-dogsquared** [comment]
Hey there! I apologize for the delays! >_<
I've been buried with stuff and I don't actively maintain this theme (at least for the time being, I've yet to push v3.0).

> so whenever I run hugo server, my site works fine, but on netlify it doesnt show any of the css

Usually, that is caused by a configuration error (most likely the `baseURL` key). Also, if possible, could you link the repo so that I check on it?

> Running hugo server also gives this error
> `WARN 2020/12/18 10:16:08 .File.Dir on zero object. Wrap it in if or with: {{ with .File }}{{ .Dir }}{{ end }}`

It's more of a warning than an error, you shouldn't have problems continuing with the build. However, it's still something I have to address since the usage is pretty ugly (in my opinion) and potentially unsafe.

Seeing as the linked website seems fine (at least a month from when this issue is filed) so I guess this is resolved? But still, I hope this clear some things up. Let me know if you still have problems and thanks for checking out this theme, btw. :)

**Comment by jgx0** [comment]
@foo-dogsquared yeah so the repo is https://github.com/unixgang/website and can you also (just as a suggestion) try to make it easier to use as a blog theme?

**Comment by jgx0** [comment]
also @foo-dogsquared we cant blog on the website, and if we do it is not on the list

**Comment by foo-dogsquared** [comment]
> can you also (just as a suggestion) try to make it easier to use as a blog theme?

I suppose reducing options would make it easier which is what I'm doing for the next major version of this theme. It also makes it easier for me by making it easier to modify for the site author. Much better than second-guessing which options does my hypothetical user needs.

> we cant blog on the website, and if we do it is not on the list

I looked at your repo and here are some of the corrections:

- In your site config (`config.toml`), remove the quotes of the value of `hidePostsOnHome` (i.e., `"false"` -> `false`). Hugo template considers an non-empty string to be equivalent to true.
- Remove the `drafts` key in the frontmatter of your contents. The theme does not render drafts. [This is universal behavior throughout the themes](https://gohugo.io/getting-started/usage/#draft-future-and-expired-content) unless you set `--buildDrafts` during the build. I guess this should be documented in the manual (maybe in the content organization).

I tried the above procedures and it worked.

![image](https://user-images.githubusercontent.com/34962634/105804643-47158000-5fdb-11eb-8073-8c8dad6ae47a.png)

**Comment by jgx0** [comment]
It worked! Thank you very much! Should I close the issue?

**Comment by foo-dogsquared** [comment]
Yeah sure, if you have no more problems.
